coil-base/src/main/java/coil/map/MeasuredMapper.kt

```diff
@@ -14,6 +14,7 @@ import coil.target.Target
  *
  * @see Mapper
  */
+@Deprecated("Superseded by `coil.interceptor.Interceptor` which offers a more versatile API.")
```

Comment:
Probably no straightforward way to do a `ReplaceWith`, eh?

Reply:
Unfortunately the replace is a little too complex for autocomplete, but a `MeasuredMapper` -> `Interceptor` conversion will end up looking like this:

```kotlin
class SizeInterceptor : Interceptor {

    suspend fun intercept(chain: Chain): RequestResult {
        val request = chain.request
        if (handles(request)) {
            val newRequest = request.newBuilder().data(map(request.data, chain.size)).build()
            return chain.proceed(newRequest)
        } else {
            return chain.proceed(request)
        }
    }
}
```